components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/AdditionalSubscriptionInfoListDTO.java

```diff
@@ -0,0 +1,128 @@
+package org.wso2.carbon.apimgt.rest.api.store.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import org.wso2.carbon.apimgt.rest.api.store.v1.dto.AdditionalSubscriptionInfoDTO;
+import org.wso2.carbon.apimgt.rest.api.store.v1.dto.PaginationDTO;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+
+import javax.xml.bind.annotation.*;
+import org.wso2.carbon.apimgt.rest.api.common.annotations.Scope;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import javax.validation.Valid;
+
+
+
+public class AdditionalSubscriptionInfoListDTO   {
+  
+    private Integer count = null;
+    private List<AdditionalSubscriptionInfoDTO> list = new ArrayList<AdditionalSubscriptionInfoDTO>();
+    private PaginationDTO pagination = null;
+
+  /**
+   * Number of additional information sets of subscription returned. 
+   **/
+  public AdditionalSubscriptionInfoListDTO count(Integer count) {
+    this.count = count;
+    return this;
+  }
+
+  
+  @ApiModelProperty(example = "1", value = "Number of additional information sets of subscription returned. ")
+  @JsonProperty("count")
+  public Integer getCount() {
+    return count;
+  }
+  public void setCount(Integer count) {
+    this.count = count;
+  }
+
+  /**
```

Comment:
Dont leave empty comments

Reply:
This file is an auto-generated DTO file. Hence we cannot change anything on this file.